package.json

```diff
@@ -131,7 +131,7 @@
     "cozy-scripts": "^6.3.8",
     "cozy-sharing": "4.1.5",
     "cozy-stack-client": "^32.2.5",
-    "cozy-ui": "^68.4.0",
+    "cozy-ui": "^70.6.2",
```

Comment:
it's not 71, it's 70

did you look at the breaking change of cozy-ui?
I think we should apply the codemods + remove @material-ui/core as dependency

Reply:
Yep this is mentioned in the TODO list from the PR, i cannot put the final commit here yet as cozy-ui's PR is still not merged

The codemods have been applied already, not Text component is used in the project. Regarding material-ui, I don't get it, the BC is not that we MUST remove it, but it is about them not being mandatory anymore. If we must remove them, maybe we should rewrite the BC?

Reply:
Yes it's not mandatory. => nit

But the maintainer of Drive will appreciate, not to have this useless dependency in the application.  😄

Reply:
As we decide changes on that import during last front meeting, I think it's better to let it go.